Objects/listobject.c

```diff
@@ -1139,12 +1139,11 @@ sortslice_advance(sortslice *slice, Py_ssize_t n)
            if (k)
 
 /* The maximum number of entries in a MergeState's pending-runs stack.
- * This is enough to sort arrays of size up to about
- *     32 * phi ** MAX_MERGE_PENDING
- * where phi ~= 1.618.  85 is ridiculouslylarge enough, good for an array
- * with 2**64 elements.
+ * For a list with n elements, thia needs at most floor(log2(n)) + 1 entries
```

Comment:
typo: "this"? instead of "thia"

Reply:
Thanks! Repaired. There are several Python implementations of the power computation in files attached to the bpo report. and the one here is a more-than-less mechanical translation of runstack.py's `powerloop()` function. I did check the C version on various cases, but agree "it would be nice" if that were an automated thing. But, it's not :wink:.